pkg/disk/disk.go

```diff
@@ -110,6 +113,41 @@ func NewFSType(s string) (FSType, error) {
 	}
 }
 
+// PartitionTableType is the partition table type enum.
+type PartitionTableType uint64
```

Comment:
I'm confused, why is this part of the displayed diff here? Is GH having a bad day? This should already be in `main`, no?

Reply:
That's my fault.  I rebased on the head of the merge queue while the other PR was queued up to see if I get any issues.  Should be good now.  Sorry for the confusion.